### PR TITLE
Fixed admin page according to comments

### PR DESCRIPTION
**Pull request by VladimirSushinsky**
## Github ticket

* [Github ticket](https://github.com/ita-social-projects/VictoryCenter-Back/issues/204)

## Description

Compare and eliminate inconsistencies between the mockup (Figma) and the implemented frontend functionality of the admin team page according:
https://discord.com/channels/1344044167570915339/1372089997036879955/1387397757802647642

## How it looks

image

image

## Summary of change

 - side panel black line width is 8px;
 - side panel is aligned with tabs;
 - side panel buttons have 60px width;
 - team member cards have 8px identation;
 - updated category dropdown;
 - text in photo upload button is centered.

## How to recreate changes

1. Go to admin page(http://localhost:3000/admin-page/team)
2. Open Add-member modal

## CHECK LIST
- [ ]  New tests was added or existing was modified
- [ ]  Changes has severe impact on users
- [ ]  Changes has medium impact on users
- [ ]  Changes has light impact on users
- [ ]  Test covetage was updated
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/common/select/Select.test.tsx [hunk 1]
```diff
@@ -146,6 +146,8 @@ describe('Select Component', () => {
         expect(span).toHaveStyle('color: #61615C');
 
         fireEvent.click(selectContainer);
+        expect(span).toHaveStyle('color: #061125');
```
Comment (Rominos7) [inline src/components/common/select/Select.test.tsx:149]: Suggestion:
For the future you can skip that kind of checks in tests

File: src/components/common/select/Select.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ export const Select = <TValue, >({
                      }
                  }}>
         <span
-            style={selectedValue !== null && selectedValue !== undefined ? {color: "#061125"} : {color: "#61615C"}}> {selectedValue !== null && selectedValue !== undefined ? selectedValue.toString() : 'Статус'}</span>
+            style={isOpen || (selectedValue !== null && selectedValue !== undefined) ? {color: "#061125"} : {color: "#61615C"}}> {selectedValue !== null && selectedValue !== undefined ? selectedValue.toString() : 'Статус'}</span>
```
Comment (Rominos7) [inline src/components/common/select/Select.tsx:42]: Two things that I can suggest:
1) Can you refactor this part to not change styles in style tag? Reason - styles in style tag would be very hard to override with class.
What can you do is use this lib [classNames](https://www.npmjs.com/package/classnames)
And refactor this part 

```
<span
  className={classNames({ 
  'color1-class': condition 1,
  'color2-class': condition 2 })}
>
{Content}
</span>
```

2) Can you exstract 'Cтатус' string into the const and import from const file? Like `const STASTUS = 'Статус'`

File: src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx [hunk 3]
```diff
@@ -82,6 +82,7 @@ const ConfirmPublishModal = ({
     <Modal isOpen={isOpen} onClose={onCancel} data-testid="publish-confirm-modal">
         <Modal.Title>{TEAM_PUBLISH_NEW_MEMBER}</Modal.Title>
         <Modal.Content>
+            <></>
```
Comment (Rominos7) [inline src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx:85]: Just a question:
Why you need to use `<></>` here?

File: src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx [hunk 4]
```diff
@@ -106,7 +107,7 @@ const ConfirmCloseModal = ({
     <Modal isOpen={isOpen} onClose={onCancel} data-testid="confirm-close-modal">
         <Modal.Title>{TEAM_CHANGES_LOST}</Modal.Title>
         <Modal.Content>
-
+            <></>
```
Comment (Rominos7) [inline src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx:110]: Same question here

File: src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx (hunk 3 above)
Comment (VladimirSushinsky) [reply src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx:85]: My IDE was giving me an error here, which was annoying. Adding <></> fixed it.

File: src/components/common/select/Select.test.tsx (hunk 1 above)
Comment (Rominos7) [reply src/components/common/select/Select.test.tsx:149]: Its more like edge case here

You do not exlicitly set class with dedicated name in element, you more like checking color value for element

Still there is no issue if you add that check

Comment (Rominos7) [reply src/components/common/select/Select.test.tsx:149]: Up to you:
Would be good if you set this color with a class and then check if that class (with dedicated name) is set for element

File: src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx (hunk 3 above)
Comment (Rominos7) [reply src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx:85]: Hmmm...
I guess it is [expects child](https://github.com/ita-social-projects/VictoryCenter-Client/blob/release/1.0.0/src/components/common/modal/Modal.tsx#L87) in `<Modal.Content>` so by adding Fragment or `<></>` basically provides "empty" child

Comment (Rominos7) [reply src/pages/admin/team/components/team-page-toolbar/TeamPageToolbar.tsx:85]: For the future (not here):
We may want to consider to refactor `Modal` component so it would not be mandatory to use `Modal.Content` with empty child

File: src/components/common/select/Select.test.tsx (hunk 1 above)
Comment (VladimirSushinsky) [reply src/components/common/select/Select.test.tsx:149]: Understood, I will fix this later when I will be integrating classNames library.

File: src/components/common/select/Select.tsx (hunk 2 above)
Comment (VladimirSushinsky) [reply src/components/common/select/Select.tsx]: Fixed as suggested.

**Review by OlyaMraka — APPROVED** [review]

File: src/components/common/select/Select.tsx [hunk 5]
```diff
@@ -39,7 +41,11 @@ export const Select = <TValue, >({
                      }
                  }}>
         <span
-            style={selectedValue !== null && selectedValue !== undefined ? {color: "#061125"} : {color: "#61615C"}}> {selectedValue !== null && selectedValue !== undefined ? selectedValue.toString() : 'Статус'}</span>
+            className={classNames('empty', {
+                'not-empty': selectedValue !== null && selectedValue !== undefined})}
```
Comment (Rominos7) [inline src/components/common/select/Select.tsx:45]: One other suggestion
If you would track span state using classes `empty/not-empty` then it will be confuthin seens those two classes together like `<span class="empty not-empty">`

It would not be clear if span `is empty or not?`

you can do next
```
const isSelectedValuePresent = !!selectedValue;

<span
  className{classNames({
     'empty': !isSelectedValuePresent,
      'not-empty': isSelectedValuePresent
  })}
>
```

in that way you would have only one class present for span and it would be good for tracking span state
